components/Main/ContactUs/index.tsx

```diff
@@ -1,8 +1,7 @@
-import Button from '../Button'
 import MessageForm from './MessageForm'
 import React, { useState } from 'react'
 import Section from '../form/Section'
-import style from './ContactUs.module.css'
+import SuccessConfirmation from '../form/SuccessConfirmation'
 
 export default function ContactUs(): JSX.Element {
```

Comment:
Wondering whether this component can be generalised and shared between this and the `NewsletterSignUp`.

Reply:
I'm with you the structure of the two pieces of code is similar. However, most attributes are different. I think the structure is clear and can self-explain now. And I'm worried if we generalize the structure to another layer, the attributes will mix together and lost their relationship. 
![image](https://user-images.githubusercontent.com/30286194/119241823-71540480-bbad-11eb-8889-4a4fbe6b980f.png)

Reply:
No this is pretty much what I meant. If I'm understanding you correctly by attributes you are talking about props? So yeah I would assume things like `title`, `description`, and the children would be different and that the structure is the abstraction. A little unsure of the renaming from `isSuccessful` to `isFilling`, what is the reasoning here?

Reply:
Ok, I will abstract the structure in a shared component.

Reply:
As to renaming, that's because I want to use true as default, and move `SignUpForm` in front of `SuccessConfirmation`. Which one do you think is better/common?

Reply:
Oh, I realize 'isFilling' is not a good name. Since `!isFilling` doesn't mean successfully submitted. I will revert this change.

Reply:
For me what makes the most sense is probably `isSubmitted` as it describes what has happened. `isFilling` seems more applicable for while the user is filling out the form and `isSuccessful` would be more like if the response from the back end was successful.

Reply:
I haven't done all updates. I'll let you know when done. I realize how forms can be tricky now.

Reply:
Beware the gerunds. Words with -ing endings generally mean something that is in progress, while -ed endings mean something that is completed.